app/components/Combobox.tsx

```diff
@@ -205,6 +213,11 @@ export function UserCombobox({
     );
   }
 
+  // Reference for Fuse options: https://fusejs.io/api/options.html
+  const USER_COMBOBOX_FUSE_OPTIONS = {
+    threshold: 0.42, // Empirically determined value to get an exact match for a Discord ID
+  };
```

Comment:
Let's move this outside the function. Doesn't really matter but it's how I've been doing it and I find it cleaner as when you reading through the function it's not often relevant what exactly the `USER_COMBOBOX_FUSE_OPTIONS` is :)